apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Trace/CloudTraceExtension.cs

```diff
@@ -116,6 +117,9 @@ public static IApplicationBuilder UseGoogleTrace(this IApplicationBuilder app)
 
             services.AddSingleton(tracerFactory);
 
+            // Only add the HttpContextAccessor if it's not already added.
```

Comment:
Maybe add a comment about why it needs to be added (so it isn't accidentally removed again).

Reply:
Good call. Done